src/components/molecules/Chatbox/Chatbox.scss

```diff
@@ -1,65 +1,63 @@
-@import "scss/constants.scss";
-
-$primary: #005ee5;
-$black: #000000;
-$white: #fff;
-$dark: #19181a;
-$border-radius: 28px;
+.chatbox-container {
+  all: unset;
```

Comment:
I have used it in some `inputs/p/buttons` to remove styles, set by global styles. Probably, not a very good idea to change/remove the global styles in this PR because it effects many places

Reply:
Can you maybe add a `@debt` that says that then, that way we can clean it up better when the global styles are removed/improved.

Something like: `// @debt Workaround to avoid styles set by src/scss/global.scss (fix this when when clean that up)`

(mentioning the filename like that will cause my IDE to pick it up as a potential usage if I tried to remove the global styles file, so will make sure we don't accidentally miss it.